tasks.py

```diff
@@ -298,3 +298,44 @@ def tests(context, name=NAME, image_ver=IMAGE_VER, local=INVOKE_LOCAL):
     pytest(context, name, image_ver, local)
 
     print("All tests have passed!")
+
+
+@task
+def html(context, sourcedir="docs/source", builddir="docs/build"):
```

Comment:
Do we need to add `docs/build` to `.gitignore`?

Reply:
Added.